sdk/servicebus/service-bus/samples/javascript/advanced/sessionRoundRobin.js

```diff
@@ -82,8 +82,8 @@ async function receiveFromNextSession(serviceBusClient) {
     });
   } catch (err) {
     if (
-      isMessagingError(err) &&
-      (err.code === "SessionCannotBeLockedError" || err.code === "OperationTimeoutError")
+      isServiceBusError(err) &&
+      (err.reason === "SessionCannotBeLocked" || err.reason === "OperationTimeout")
```

Comment:
So, we'll ask users to look for reason and not code. Isn't it inconsistent with event-hubs?

Reply:
This is fine for the preview, lets continue this discussion for GA

Reply:
I don't think it's inconsistent, btw. code is a wide-open field vs reason which is curated. This kind of 'raw value' vs 'interpreted value' isn't a bad pattern (IMO).